### ISSUE TITLE
Missing search icon

### ISSUE THREAD

**Issue by rchl** [SublimeText/Theme-DAneo#4]
With `Elegant` theme variant (and some other) `Goto anything` input should have a search icon (according to images in readme), but I can't see any here. Only reserved space.

![Screenshot 2019-07-10 at 22 14 34](https://user-images.githubusercontent.com/153197/61001557-2259a900-a360-11e9-989a-753ee7250ec2.png)

**Comment by deathaxe** [comment]
Which OS and display resolution and scale do you use?

I tried on Windows with different ui_scales. Maybe some settings are interpreted slightly different on Linux/MacOS, which cause the required space for the bitmap to be too small.

**Comment by rchl** [comment]
MacOS, 2560x1440, no scaling (or default but I think there is no scaling in that resolution).

It works if I increase `content_margin` `top` value from 9 to 11.

```json
		{
			"class": "text_line_control",
			"parents": [
				{"class": "overlay_control"}
			],
			"content_margin": [52, 11, 13, 13],
			"color_scheme_tint": "var(overlay_input_tint-dark)",
			"layer0.draw_center": false,
			"layer0.inner_margin": [52, 48, 0, 0],
			"layer0.texture": "Theme - DAneo/textures/inputs/input_search.png",
			"layer0.tint": "var(accent)",
			"layer0.opacity": 1.0,
			"layer1.texture": "",
			"layer1.opacity": 0.0
		},
```

**Comment by rchl** [comment]
So I guess whether bug triggers depends on actual height of the text. Height of the text + margins must be big enough to fit image. I don't know themes format enough to know how to make this more flexible.

**Comment by deathaxe** [comment]
Yes it does. I can remember this issue during development. If ST really interprets margins/positions differnently on different OSs, I wonder whether there are other edge cases which lead to some quirks.

**Comment by deathaxe** [comment]
Should be fixed by v0.2.0

**Comment by rchl** [comment]
Yes, it is. Thanks. :)
